itests/util/src/main/java/org/apache/hadoop/hive/cli/control/CliConfigs.java

```diff
@@ -236,7 +240,39 @@ public MiniLlapLocalCliConfig() {
       }
     }
   }
+  
+  public static class MiniLlapLocalCompactorCliConfig extends AbstractCliConfig {
+
+    public MiniLlapLocalCompactorCliConfig() {
+      super(CoreCliDriver.class);
+      try {
+        setQueryDir("ql/src/test/queries/clientpositive");
+
+        includesFrom(testConfigProps, "compaction.query.files");
+        setResultsDir("ql/src/test/results/clientpositive/llap");
+        setLogDir("itests/qtest/target/qfile-results/clientpositive");
+
+        setInitScript("q_test_init.sql");
+        setCleanupScript("q_test_cleanup.sql");
+
+        setHiveConfDir("data/conf/llap");
+        setClusterType(MiniClusterType.LLAP_LOCAL);
+        setCustomConfigValueMap(createConfVarsStringMap());
+      } catch (Exception e) {
+        throw new RuntimeException("can't construct cliconfig", e);
+      }
+    }
 
+    private static Map<HiveConf.ConfVars, String> createConfVarsStringMap() {
+      Map<HiveConf.ConfVars,String> customConfigValueMap = new HashMap<>();
+      customConfigValueMap.put(HiveConf.ConfVars.HIVE_EXECUTION_ENGINE, "tez");
```

Comment:
1. Why do we need to explicitly set the exec engine? isn't it already TEZ
2. What if we want to create tests for the MR compaction, I think that one should stay default. In q-tests, we can change to query-based. WDYT?

Reply:
1. It's true it should be TEZ already from config.
2. Ok, deleting this.